### DISCUSSION TITLE
How i can show the Direct parent folder name/title inside my Search Results

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3599] [Q&A]
I have added this PnP Search Results, to only show folders with name ="Specs" inside a specific document library :-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/d9b4d3d1-4cd4-4bae-b3d6-21fe19c9be66)

and we got this result:-

![image](https://github.com/microsoft-search/pnp-modern-search/assets/79726511/252ad1c2-6283-4808-aaaf-4c3bc84b3661)

Our folder structure looks as follow:-

Documents
   1) CustomerA
        -  Main
        -  General
        -  Specs 
   2) CustomerB
        -  Main
        -  General
        -  Specs 
   3) CustomerC
        -  Main
        -  General
        -  Specs 

so currently inside the Search results it will only show the sub-folder names, but we need to know to which parent folder (customerA or customerB or CustomerC) this "Specs" folder belongs.. is there a way to do this?

Thanks

**Comment by wobba** [comment]
You can requesr to have the ParentLink managed property returned. It's not the real title of the parent, but the part after the last / will be the folder URL name for items in a folder. Use handlebar helpers functions to split and take the last part.

**Reply by mvcsharepointdev** [reply]
@wobba can you provide a sample code or screen shot of what i need to do exactly ? Thanks

**Comment by wobba** [comment]
Add property name and hit return. Then it will appear in the list.
<img width="308" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/51104/76ed5a9e-84b8-4617-aa93-67d82668a855">

Handlebar to get the Folder part of the parent link `{{itemAt (split ParentLink "/")  (subtract (length (split ParentLink "/")) 2)}}` which can be used in a card filed if you use the card layout.
